backend/src/main/routes/ProjectRoute.ts

```diff
@@ -0,0 +1,76 @@
+import express from 'express';
+import { CommonRoutesConfig } from './CommonRoutesConfig';
+import { ReasonPhrases, StatusCodes } from 'http-status-codes';
+import { inject, injectable } from 'tsyringe';
+import { ProjectService } from '../services/ProjectService';
+import HttpException from '../exceptions/HttpException';
+import { WorksonService } from '../services/WorksOnService';
+
+@injectable()
+export default class ProjectRoute extends CommonRoutesConfig {
+  constructor(
+    @inject('express-app') app: express.Application,
+    private projectService: ProjectService,
+    private worksOn: WorksonService
+  ) {
+    super(app, 'ProjectRoute');
+  }
+
+  configureRoutes(): express.Application {
+    this.getApp()
+      .route(`/project`)
+      .post(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+          const newProject = await this.projectService.createProject(req.body);
+          const dto = JSON.parse(JSON.stringify(newProject));
+          res.status(StatusCodes.CREATED).send(dto);
+        } catch (err) {
+          next(err);
+        }
+      })
+      .get(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+          const projectOfBusiness = await this.projectService.getProjectofBusiness(Number(req.query.businessId));
+          res.status(StatusCodes.OK).send(projectOfBusiness);
+        } catch (err) {
+          next(err);
+        }
+      });
+
+    this.getApp()
+      .route(`/project/:id`)
+      .get(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+          const project = await this.projectService.getProject(Number(req.params.id));
+          const employees = await this.worksOn.getWorksOn(Number(req.params.id));
+          const array = [];
```

Comment:
you can combine line 46 to 48 together by doing `const array = [project, employees]`

Reply:
ok good